### PR TITLE
Home

### PR DESCRIPTION
**Pull request by edubew**
## In this milestone the following was done:

- [x]   Create react app
- [x]   Build files structure for the React app.
- [x]   Prepare routes and navigation in the react app.
- [x]   Select the API endpoint for react app data fetch.
- [x]   Display a List of items and filtered items data fetched from the API.
- [x]   Add details link for all fetch data.
- [x]   Create tests for the react application.
- [x]   Add style for every component
- [x]   Add unit test
- [x]   Deploy the project
- [x]   Create a professional README file

## [Video Presentation](https://www.loom.com/share/87c2364a87a047da9807678aadbc0c7f)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,229 @@
...
 
-### Code Splitting
+<!-- LIVE DEMO -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## 🚀 Live Demo <a name="live-demo"></a>
 
-### Analyzing the Bundle Size
+- [Live Demo Link](https://virtual-currency.onrender.com)
+- [Video explaination Link]()
```
Comment (gemmen29) [inline README.md]: - [ ] Kindly add a video link as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Once%20you%20have,very%20proud%20of.).

File: src/Tests/CoinDetails.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```
Comment (gemmen29) [inline src/Tests/CoinDetails.test.js:1]: - Good job on implementing the snapshots test for components. but, 
   - [x] Kindly add unit tests for pure functions in your Redux code as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.).
   - [x] Please add integration tests for your application using the React Testing Libraryas per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests,the%20Redux%20Store.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @edubew, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @edubew 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ All tests pass
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**